notebook/tests/selenium/utils.py

```diff
@@ -100,6 +100,16 @@ def focus_cell(self, index=0):
         self.to_command_mode()
         self.current_cell = cell
 
+    def find_and_replace(self, index=0, find_txt='', replace_txt='', replace_all=False):
```

Comment:
Looks like the `replace_all` parameter doesn't do anything. It's fine to drop it for now - it can always be added back when there's a test that wants to use it.